site/src/templates/standard-article.js

```diff
@@ -187,6 +210,8 @@ export default class Article extends React.Component {
                   </span>
                 )
               }
+              roles={roles}
+              isArticle
```

Comment:
Why is this needed on something called "standard-article.js"?  In what circumstances is this template used for something that is not an article? This smells odd.